cmd/maya-apiserver/cstor-operator/spc/handler.go

```diff
@@ -174,7 +175,7 @@ func validatePoolType(spc *apis.StoragePoolClaim) error {
 // validateDiskType validates the disk types in spc.
 func validateDiskType(spc *apis.StoragePoolClaim) error {
 	diskType := spc.Spec.Type
-	if !(diskType == string(apis.TypeSparseCPV) || diskType == string(apis.TypeDiskCPV)) {
+	if !(diskType == string(apis.TypeSparseCPV) || diskType == string(apis.TypeDiskCPV) || diskType == string(apis.TypeBlockDeviceCPV)) {
```

Comment:
how about having check only for BD? Disk or sparse should not be used

Reply:
Thinking the same and got confirmation in slack channel. I will update the check